plugins/flytekit-flyin/setup.py

```diff
@@ -15,7 +15,7 @@
     author_email="admin@flyte.org",
     description="This package holds the flyin plugins for flytekit",
     namespace_packages=["flytekitplugins"],
-    packages=[f"flytekitplugins.{PLUGIN_NAME}"],
+    packages=[f"flytekitplugins.{PLUGIN_NAME}", f"flytekitplugins.{PLUGIN_NAME}.vscode_lib"],
```

Comment:
now that we have more than one package in this plugin, can we use setuptools [find_packages](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html) instead of listing them exhaustively here?

Reply:
Yes, I can try it!

Reply:
update: I will do this in a new PR.